lib/layout.js

```diff
@@ -126,20 +138,43 @@ const PodiumLayout = class PodiumLayout {
     async process(incoming) {
         const s = incoming;
         s.name = this.name;
+        s.view = this._view;
+        s.development = true;
 
         const s1 = await this.context.process(s);
         const s2 = await this.httpProxy.process(s1);
         return s2;
     }
 
+    view(fn = null) {
+        if (!putils.isFunction(fn)) {
+            throw new Error(
+                `Value on argument variable "template" must be a function`,
+            );
+        }
+        this._view = fn;
+    }
+
+    render(incoming, data) {
+        return incoming.render(data);
+    }
+
     middleware() {
         return (req, res, next) => {
             const incoming = new HttpIncoming(req, res, res.locals);
 
             this.process(incoming)
                 .then(result => {
                     if (result) {
-                        putils.setAtLocalsPodium(res, 'context', incoming.context);
+                        putils.setAtLocalsPodium(
+                            res,
+                            'context',
+                            incoming.context,
+                        );
+
+                        res.podiumSend = data =>
+                            res.send(this.render(incoming, data));
+
```

Comment:
Should we unroll this promise chain to a `async`/`await`?

Reply:
yup